cmake/version.cmake

```diff
@@ -0,0 +1,14 @@
+find_package(Git QUIET)
+
+# default version string
+set(LIBGRAPHQLPARSER_VERSION "0.0-dev")
```

Comment:
the current release is v0.5.0 -- https://github.com/graphql/libgraphqlparser/releases